src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Truck start work");

```

Comment:
The message in the `doWork` method should be more informative and indicate the specific type of machine that is working. For example: 'Truck is doing its work.'